### PR TITLE
Feature/sim player scripts changelog

### PR DESCRIPTION
**Pull request by MatthewSandfordImprobable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

#### Description
Adding changelog note for the new scripts being added to UnrealExampleProject.

#### Primary reviewers
@oblm @m-samiec

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The server no longer crashes, when received RPCs are processed recursively.
 - DeploymentLauncher can parse a .pb.json launch configuration.
 
+### Features:
+- Added two scripts to the UnrealExampleProject: LaunchSimPlayerClient.bat and LaunchSimPlayers.bat. LaunchSimPlayerClient.bat will launch a single SimPlayer client. LaunchSimPlayers.bat will launch 10 SimPlayer clients.
```
Comment (oblm) [inline CHANGELOG.md]:
```suggestion
- In local deployments of the Example Project you can now launch Simulated Players in one click. Running `LaunchSimPlayerClient.bat` will launch a single Simulated Player client. Running `LaunchSimPlayers.bat` will launch 10.
```

**Comment by oblm** [comment]
If you accept [this suggestion](https://github.com/spatialos/UnrealGDKExampleProject/pull/79#issuecomment-559046077) be sure to rename the script in this CHANGELOG entry

File: CHANGELOG.md (hunk 1 above)
Comment (oblm) [reply CHANGELOG.md]: This better conforms to the WIIFM format advocated in: [how-to-write-good-release-notes.md](https://github.com/spatialos/UnrealGDK/blob/release/SpatialGDK/Extras/internal-documentation/how-to-write-good-release-notes.md).

Comment (MatthewSandfordImprobable) [reply CHANGELOG.md]: Ooooh, nice.

**Pull request merged**